### ISSUE TITLE
UPP: error for small query files

### ISSUE THREAD

**Issue by smirarab** [smirarab/sepp#10]
If the number of query sequences is small (only one in my test case), you can get empty fragment chunk files, which causes and error:

```
[08:54:11] jobs.py (line 66):     INFO: Starting hmmsearch Job with input: model:/tmp/sepp/FAA.R9LRK1/root/P_0/A_0_1/hmmbuild.model.XF1w8u, fragments:/tmp/sepp/FAA.R9LRK1/fragment_c
hunks/fragment_chunk_1VfXGJI.fasta, elim:99999999, filter:False, output:Piped
[08:54:11] jobs.py (line 117):     INFO: Finished hmmsearch Job with input: model:/tmp/sepp/FAA.R9LRK1/root/P_0/A_0_1/hmmbuild.model.XF1w8u, fragments:/tmp/sepp/FAA.R9LRK1/fragment_
chunks/fragment_chunk_1VfXGJI.fasta, elim:99999999, filter:False, output:Piped with:
 return code: 1
 output:
 error:
Error: Sequence file /tmp/sepp/FAA.R9LRK1/fragment_chunks/fragment_chunk_1VfXGJI.fasta is empty or misformatted
```

**Comment by smirarab** [comment]
fixed with 12602464416dde842e65e8dfe80a9592dd314085

**Comment by namphuon** [comment]
Sorry, was in meetings all day, just finished.  Thanks for fixing that for
me.

On Wed, Mar 18, 2015 at 11:48 AM, Siavash Mirarab <notifications@github.com>
wrote:

> Closed #10 <https://github.com/smirarab/sepp/issues/10>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/smirarab/sepp/issues/10#event-258362961>.
>



-- 
Nam-phuong Nguyen, PhD
Carl R. Woese Institute for Genomic Biology,
University of Illinois, Urbana-Champaign
1206 W. Gregory Drive, Urbana, IL 61801
